.travis.yml

```diff
@@ -0,0 +1,44 @@
+# trusty beta image has jdk8, gcc4.8.4
+dist: trusty
+sudo: required
+# xcode8 has jdk8
+osx_image: xcode8
+# Not technically required but suppresses 'Ruby' in Job status message.
+language: java
+
+os:
+  - linux
+  - osx
+
+env:
+  - BAZEL=0.22.0
+
+before_install:
+  - |
+    if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
+      OS=darwin
+    else
+      sysctl kernel.unprivileged_userns_clone=1
+      OS=linux
+    fi
+    wget -O install.sh "https://github.com/bazelbuild/bazel/releases/download/${BAZEL}/bazel-${BAZEL}-installer-${OS}-x86_64.sh"
+    chmod +x install.sh
+    ./install.sh --user
+    rm -f install.sh
+
+before_script:
+  - echo "startup --output_base=$HOME/.cache/bazel" >> .bazelrc
+  - # This is from Bazel's former travis setup, to avoid blowing up the RAM usage.
+  - echo "startup --host_jvm_args=-Xmx2500m" >> .bazelrc
+  - echo "startup --host_jvm_args=-Xms2500m" >> .bazelrc
```

Comment:
Is it really needed?

Reply:
I have no idea. Without this configuration it works perfectly fine - see the build https://travis-ci.org/realityforge/elemental2/jobs/489272694 for an example where the configuration was not present.

However after looking at how several other projects that use TravisCI and bazel together I noticed that they all had it so I cargo culted it in.

I will remove it and if it ever becomes an issue then we can add it back in